Src/Actions/DisplayMessageAction.lua

```diff
@@ -0,0 +1,42 @@
+local TextService = game:GetService("TextService")
+
+local Action = require(script.Parent.Parent.API.Action)
+
+return function()
+	local DisplayMessageAction = Action.new({
+		Name = "Display Message Action",
+		Uuid = "internal:display-message-action",
```

Comment:
So on the server if it starts with internal: i should ignore it?

Reply:
It's `:internal` because it's an internal action defined by the SDK, the end developer won't be writing this action so it's not external in that sense 🤔